packages/flutter/lib/src/services/raw_keyboard_linux.dart

```diff
@@ -39,32 +39,30 @@ class RawKeyEventDataLinux extends RawKeyEventData {
   /// (GLFW, GTK, QT, etc) may have a different key code mapping.
   final KeyHelper keyHelper;
 
+  /// The name of the printable key, encoded as UTF-8. This is
+  /// typically the character that key would produce without any modifier keys.
+  /// For dead keys, it is typically the diacritic it would add to a character.
+  final String charactersIgnoringModifiers;
+
   /// The hardware scan code id corresponding to this key event.
   ///
   /// These values are not reliable and vary from device to device, so this
   /// information is mainly useful for debugging.
   final int scanCode;
 
-  /// The Unicode code point represented by the key event, if any.
-  ///
-  /// If there is no Unicode code point, this value is zero.
-  ///
-  /// Dead keys are represented as Unicode combining characters.
-  final int codePoint;
-
   /// The hardware key code corresponding to this key event.
   ///
   /// This is the physical key that was pressed, not the Unicode character.
-  /// See [codePoint] for the Unicode character. This value may be different depending
-  /// on the window toolkit used (See [toolkit]).
+  /// See [charactersIgnoringModifiers] for the unmodified Unicode character. This value may be different
+  /// depending on the window toolkit used (See [toolkit]).
```

Comment:
```suggestion
  /// depending on the window [toolkit] used.
```
All of the `(See [toolkit)`s at the end of these properties could probably be reduced to just the breadcrumb in these mentions and broken out into a `See Also: ...`

Reply:
It was actually the wrong reference, it should've pointed to KeyHelper :) Fixed